src/index.js

```diff
@@ -34,17 +34,55 @@ function createTask() {
   }
   const task = new Todo(text);
 
+  updatePosition(tasks);
   tasks.push(task);
+
   localStorage.setItem('tasks', JSON.stringify(tasks));
   renderTasks(tasks);
 
   addInput.value = '';
 }
 
+const saveList = (list) => {
+  window.localStorage.setItem('tasks', JSON.stringify(list));
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+};
+
+export default function editItem(event, index) {
+  tasks.forEach((task) => {
+    if (task.index === Number(index)) {
+      task.description = event.target.value;
+    }
+  });
+  saveList(tasks);
+}
+
 addButton.addEventListener('click', createTask);
 
 addInput.addEventListener('keypress', (event) => {
   if (event.key === 'Enter') {
     createTask();
   }
 });
+
+export  function deleteTask(indx) {
+  const newTasks = tasks.filter((task) => task.index !== Number(indx));
+
+  updatePosition(newTasks);
+  renderTasks(newTasks);
+  saveList(newTasks);
+}
+
+// function for delete all completed
+function deleteCompleted() {
+  let tasks = JSON.parse(localStorage.getItem('tasks'));
+  const newList = tasks.filter((task) => task.completed === false);
+
+  updatePosition(newList);
+  renderTasks(newList);
+  saveList(newList);
+}
```

Comment:
- [ ] Well done on implementing the functionalities. However, as mentioned in the project's requirements you have to create a new file for the new functionality. Using modules will help make the code clean and easy to maintain. Kindly reorganize your code.

Reply:
can explain more please because I think that I already did it in render and storage files

Reply:
Sorry my bad  you can request for a new review

Reply:
Thank you